components/message_queue/src/kafka/kafka_impl.rs

```diff
@@ -142,7 +142,7 @@ impl KafkaImplInner {
         }
 
         let mut client_builder =
-            ClientBuilder::new(dbg!(config.client.boost_brokers.clone().unwrap()));
+            ClientBuilder::new(config.client.boost_brokers.clone().unwrap());
```

Comment:
https://rust-lang.github.io/rust-clippy/master/index.html#/dbg_macro

Enable this clippy rule.

Reply:
Done.